src/Appwrite/Utopia/Response.php

```diff
@@ -341,6 +345,8 @@ public function __construct(SwooleHTTPResponse $response)
             ->setModel(new UsageFunctions())
             ->setModel(new UsageFunction())
             ->setModel(new UsageProject())
+            ->setModel(new Template())
```

Comment:
What is template vs emailTemplate?

Reply:
Can we have template as an abstract and extend it with both SMS and Email to make the architecture better describe our behavior?

Reply:
update

Reply:
In other models we have the namespace before the name. Ex TemplateEmail and TemplateSMS instead of EmailTemplate and SMSTemplate. Let's keep consistency.